ios/NewExpensify.xcodeproj/project.pbxproj

```diff
@@ -1848,6 +1848,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				CXX = "";
+				DEBUG_INFORMATION_FORMAT = dwarf;
```

Comment:
Odd variable 😄 


Is there no benifit to having `dSYM` enabled in dev? Does this make less logs available, for example?

Reply:
I think dSYM files doesn’t provide any benefit in dev environment. Their primary purpose is to symbolicate crash reports, which is crucial for diagnosing issues in production. During development we already have direct access to the source code, allowing us to debug without symbolicated crash reports

Reply:
Thanks, makes sense